src/main/java/rapgod/tasks/Deadline.java

```diff
@@ -0,0 +1,23 @@
+package rapgod.tasks;
+
+import rapgod.utils.Parser;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task{
+    protected LocalDateTime due;
```

Comment:
Use nouns for variables. 'due' sounds like an adjective.
```suggestion
    protected LocalDateTime dueDate;
```